app/assets/javascripts/util/search.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function searchPub(e){
+  var inputText = e?e.target.value:'';
+  var allElements = document.querySelectorAll('.leftbar-publisher');
+  if(inputText.length > 2){
+    var targetElements = document.querySelectorAll('a[data-publisher*='+inputText+']');
+    allElements
+      .forEach(function(el){
+        el.style.display = "none";
+      })
+    targetElements
+      .forEach(function(el){
+        el.parentElement.style.removeProperty('display');
```

Comment:
Instead of removing the property, can't we set it a `block` or `initial`? Removing the property will force the browser to calculate CSSOM again no?

Reply:
Fair point.